microsoft/runbook/tiers/t0.yml

```diff
@@ -1,3 +1,3 @@
 testcase:
   - criteria:
-      priority: 0
+      name: $(test_name)
```

Comment:
Please revert changes to the default tier files.

Reply:
Yes will revert all the dev setup I have. Still just WIP to document some of these things and share with other devs from our team who may also work on LISA tests. Would be good if we could document some of these tricks/available parameters for runbooks. (or maybe I missed it?)